### PR TITLE
Fix errors in documentation.

### PR DESCRIPTION
**Pull request by paulidale**
Fix some spelling errors in the documentation.
Also fix some clumsy wording.

- [x] documentation is added or updated

See #6543

### CODE REVIEW

File: doc/man3/SSL_CTX_sess_set_get_cb.pod [hunk 1]
```diff
@@ -43,9 +43,9 @@ L<SSL_CTX_set_session_cache_mode(3)>).
 (SSL/TLS server only.)
 
 SSL_CTX_sess_get_new_cb(), SSL_CTX_sess_get_remove_cb(), and
-SSL_CTX_sess_get_get_cb() allow to retrieve the function pointers of the
-provided callback functions. If a callback function has not been set,
-the NULL pointer is returned.
+SSL_CTX_sess_get_get_cb() allow retrieval of the function pointers
+provided to the associated set callback functions. If a callback function
```
Comment (levitte) [inline doc/man3/SSL_CTX_sess_set_get_cb.pod]: Shouldn't "provided" be removed?  The sentence doesn't make sense to me for the moment.

File: doc/man3/SSL_CTX_set_cert_verify_callback.pod [hunk 2]
```diff
@@ -34,10 +34,10 @@ argument I<arg> is specified by the application when setting I<callback>.
 I<callback> should return 1 to indicate verification success and 0 to
 indicate verification failure. If SSL_VERIFY_PEER is set and I<callback>
 returns 0, the handshake will fail. As the verification procedure may
-allow to continue the connection in case of failure (by always returning 1)
-the verification result must be set in any case using the B<error>
-member of I<x509_store_ctx> so that the calling application will be informed
-about the detailed result of the verification procedure!
+allow continuation of the connection in the case of failure (by always
```
Comment (levitte) [inline doc/man3/SSL_CTX_set_cert_verify_callback.pod]: Perhaps "allow the connection to continue in the case of failure"?

Comment (paulidale) [reply doc/man3/SSL_CTX_set_cert_verify_callback.pod]: Much better, thanks.

File: doc/man3/SSL_CTX_sess_set_get_cb.pod (hunk 1 above)
Comment (paulidale) [reply doc/man3/SSL_CTX_sess_set_get_cb.pod]: I think it is worse without the provided.  I've reworded differently.

**Comment by paulidale** [comment]
@levitte confirmation after the updates?

File: doc/man3/SSL_CTX_sess_set_get_cb.pod [hunk 3]
```diff
@@ -43,9 +43,9 @@ L<SSL_CTX_set_session_cache_mode(3)>).
 (SSL/TLS server only.)
 
 SSL_CTX_sess_get_new_cb(), SSL_CTX_sess_get_remove_cb(), and
-SSL_CTX_sess_get_get_cb() allow to retrieve the function pointers of the
-provided callback functions. If a callback function has not been set,
-the NULL pointer is returned.
+SSL_CTX_sess_get_get_cb() retrieve the function pointers set using the
```
Comment (levitte) [inline doc/man3/SSL_CTX_sess_set_get_cb.pod]: s/set using/set by/ ?

Comment (paulidale) [reply doc/man3/SSL_CTX_sess_set_get_cb.pod]: I'm okay with either so I'll change to _set by_

**Comment by paulidale** [comment]
Again? :)

**Comment by paulidale** [comment]
merged, thanks

**Pull request closed without merging**